plugins/grpc/swift-protobuf/v1.0.0/buf.plugin.yaml

```diff
@@ -0,0 +1,32 @@
+version: v1
+name: buf.build/grpc/swift-protobuf
+plugin_version: v1.0.0
+source_url: https://github.com/grpc/grpc-swift-protobuf
+integration_guide_url: https://swiftpackageindex.com/grpc/grpc-swift/2.0.0/documentation/grpccore
+description: Generates Swift client and server stubs for the gRPC framework.
+deps:
+  # https://github.com/grpc/grpc-swift-protobuf/blob/1.0.0/Package.swift#L42
+  - plugin: buf.build/apple/swift:v1.28.1
+output_languages:
+  - swift
+registry:
+  swift:
+    deps:
+      - source: https://github.com/grpc/grpc-swift-protobuf.git
+        package: grpc-swift-protobuf
+        # https://github.com/grpc/grpc-swift-protobuf/blob/1.0.0/Package.swift#L22
+        products: [ GRPCProtobuf ]
+        version: 1.0.0
+        # https://github.com/grpc/grpc-swift-protobuf/blob/1.0.0/Package.swift#L47
+        swift_versions: [ ".v6" ]
+        # https://github.com/grpc/grpc-swift-protobuf/blob/1.0.0/Package.swift#L123-L126
+        platforms:
+          macos: v15
+          ios: v18
+          tvos: v18
+          watchos: v11
+  opts:
```

Comment:
Just calling out that these are (assumed deliberate) deviations from the defaults here: https://swiftpackageindex.com/grpc/grpc-swift-protobuf/1.0.0/documentation/grpcprotobuf/generating-stubs#Generator-options

Reply:
yeah, I had copied these options over from [the latest version of grpc/swift](https://github.com/bufbuild/plugins/blob/551404ca2f2714eee90c0cd2c1bfd714bccef6a2/plugins/grpc/swift/v1.24.2/buf.plugin.yaml#L18-L20), but I ought to check that they work the same way in the new plugin. Will check on that before merge, thanks!

Reply:
Just took a quick peek: I believe we need `Visibility=Public` since we're publishing a package intended to be consumed by other packages. In terms of `FileNaming=PathToUnderscore`, I see in apple/swift's plugin docs that [this option can help for publishing with SPM](https://github.com/apple/swift-protobuf/blob/d72aed98f8253ec1aa9ea1141e28150f408cf17f/Documentation/PLUGIN.md?plain=1#L101-L103), which makes sense for what we're doing. (I didn't see any discussion in the [original PR that added those options](https://github.com/bufbuild/plugins/pull/508), so assuming that's the point.)

Thanks for calling out, good to check!